app/frontend/src/components/UI/sprite/Sprite.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-webpack-loader-syntax */
+import React from 'react';
+
+/**
+ * Icons must be imported separately because some of them may not be used in application
+ * otherwise try to use use following syntax:
+ */
+// eslint-disable-next-line import/no-webpack-loader-syntax
+// import icons from '!!raw-loader!icons/*.svg';
+
+import HomeIcon from '!!raw-loader!icons/home.svg';
+import DefaultAvatar from '!!raw-loader!icons/DefaultAvatar.svg';
+import PlusIcon from '!!raw-loader!icons/plus.svg';
+import SettingsIcon from '!!raw-loader!icons/settings.svg';
+
+const icons = {
+  'home': HomeIcon,
+  'DefaultAvatar': DefaultAvatar,
```

Comment:
Can we use camel case here?
```suggestion
  'defaultAvatar': DefaultAvatar,
```

Reply:
@ilyamore88 so, it should be changed where it calls too:
`<Icon name="DefaultAvatar">` or `<Button icon="DefaultAvatar" ...>` (I cant tell now where exacly it was mentioned)

We can do that in separate PR

Reply:
I wrote about it cause we have `home` and `DefaultAvatar`. So I want to use the same style. You can change it in a separate Pr.